src/app/components/room/room.service.js

```diff
@@ -144,7 +147,14 @@
             ActionService.enterRoom(msg.id, username, connection);
             // Step 3. Establish WebRTC connection with the Janus server
             // Step 4a (parallel with 4b). Publish our feed on server
+
             connection.publish({
+              success: function() {
```

Comment:
instead of abusing `success` to activate the audio after having connected muted (quite tricky), I would use the new option explained in the other comment. Something like
```javascript
var startMuted = (msg.publishers instanceof Array) && msg.publishers.length > joinUnmutedLimit;
connection.publish({
    muted: startMuted,
     error: function() { connection.publish({noCamera: true, muted: startMuted}); }
});